pontoon/base/models.py

```diff
@@ -2762,8 +2814,45 @@ def latest_activity(self):
     def __str__(self):
         return self.string
 
+    def save_stats_diff(self, **stats_diff):
```

Comment:
I would make this a method on `AggregatedStats` class and call it `adjust_all_stats()`.